### PR TITLE
Create tool get pronvinces

### PR DESCRIPTION
**Pull request by TuanLamAnh**

### CODE REVIEW

**Comment by LongDuongBIT** [comment]
Don't nest more than 2 foreach loops and reduce database calls as much as possible.

File: VMS/Startup.cs [hunk 1]
```diff
@@ -6,6 +6,8 @@
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
+using VMS.Application.Interfaces;
```
Comment (LongDuongBIT) [inline VMS/Startup.cs:9]: Please revert this file.

File: VMS/Program.cs [hunk 2]
```diff
@@ -1,19 +1,29 @@
...
 {
     public class Program
     {
-        public static void Main(string[] args)
+        public static async Task Main(string[] args)
         {
-            CreateHostBuilder(args).Build().Run();
+            var host = CreateHostBuilder(args).Build();
+
+            var startupTasks = host.Services.GetServices<IAddressPathService>();
```
Comment (LongDuongBIT) [inline VMS/Program.cs:20]: Please use `host.Services.GetRequiredService<IAddressPathService>();` instead

File: VMS.Application/Services/AddressPathService.cs [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+                AddressPathsRecursive(addressPaths, addressPathTypes, null, new Division() { Name = a.Name, DivisionType = a.DivisionType, Paths = a.Paths });
+            }
+            await _repository.InsertAsync<AddressPathType>(dbContext, addressPathTypes);
+            await _repository.InsertAsync<AddressPath>(dbContext, addressPaths);
+        }
+
+        private void AddressPathsRecursive(List<AddressPath> addressPaths, List<AddressPathType> addressPathTypes, AddressPath parentAddressPath, Division addressPathResponse)
+        {
+            TextInfo textInfo = new CultureInfo("vi-VN", false).TextInfo;
+            string divisionType = textInfo.ToTitleCase(addressPathResponse.DivisionType);
```
Comment (LongDuongBIT) [inline VMS.Application/Services/AddressPathService.cs:47]: Please make `textInfo.ToTitleCase` an extension method for later use.

**Review by LongDuongBIT — COMMENTED** [review]
Please fix those comments in advance!

File: VMS.Application/Services/AddressPathService.cs (hunk 3 above)
Comment (LongDuongBIT) [reply VMS.Application/Services/AddressPathService.cs]: Extension method please.

**Review by hoaitran308 — APPROVED** [review]
good job!

**Review by LongDuongBIT — APPROVED** [review]

**Pull request merged**